topology/Georgia Institute of Technology/Georgia Tech/Georgia Tech ISyE.yaml

```diff
@@ -0,0 +1,26 @@
+GroupDescription: Resources in the Georgia Tech PACE cluster, to be used by LIGO.
```

Comment:
I think this may describe Memo's group and resources at GT?

Reply:
I fixed the group description. Do we want a site per department?